db/migrate/20240403123520_add_lost_attribute_to_students.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddLostAttributeToStudents < ActiveRecord::Migration[7.1]
+  def change
+    add_column :students, :lost, :boolean, null: false, default: false
```

Comment:
Je nommerait la colonne plus explicitement, genre "has_data_error" ou un truc du genre.

Là ça fait un peu "enfant perdu attend sa maman à la caisse numéro 89"

Reply:
j'entends mais du point de vue de l'application c'est un élève qu'on a effectivement perdu de vue d'ailleurs 404 = not found, vs 500x = errors donc je préfère `lost` qui est simple et clair. Je peux te proposer `api_lost` même si nos élèves venants tous de l'API ça me paraît redondant.

Reply:
alors dans le contexte d'un gamin _là pour le coup_ je trouve ça limite 😄 

Après l'API c'est pas un parent si ? c'est The Source® ?

discuss

Reply:
bon j'assume mon choix, on pourra faire un renommage du booléan par une douce soirée d'été